cert-manager-configs/.applier/group_vars/seed-hosts.yml

```diff
@@ -0,0 +1,32 @@
+---
+
+openshift_domain: "my.cluster.example.com"
+cert_manager_namespace: "cert-manager"
+
+cert_manager_params:
+  DNS_DOMAIN: "{{ openshift_domain }}"
+  AWS_ACCESS_KEY_UD: "ASDF321ADF63"
```

Comment:
Typo? Should be `AWS_ACCESS_KEY_ID` I assume.

Reply:
yes, fixed.